src/current/v23.1/advanced-changefeed-configuration.md

```diff
@@ -33,6 +33,39 @@ kvadmission.rangefeed_catchup_scan_elastic_control.enabled
 
+### Improving latency
+
+Latency in changefeeds is the sum of the time it takes to:
+
+- Commit writes to the database.
+- Encode [changefeed messages]({% link {{ page.version.version }}/changefeed-messages.md %}).
+- Deliver the message to the [sink]({% link {{ page.version.version }}/changefeed-sinks.md %}).
+
+To improve changefeed latency, you can use the following [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) to reduce bursts of rangefeed work so that updates are paced steadily over time. It is necessary to adjust these cluster settings together in order to have an effect on changefeed latency. However, note that these can have an effect on running transactions, and should be adjusted and tested with your particular workload.
+
+#### `kv.closed_timestamp_target_duration`
+
+Each [range]({% link {{ page.version.version }}/architecture/overview.md %}#range) in CockroachDB tracks a property called the [_closed timestamp_]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#closed-timestamps), which means that no new writes can ever be introduced at or below that timestamp. Decreasing the target duration can reduce changefeed latency, so that the changefeed job can [checkpoint]({% link {{ page.version.version }}/how-does-an-enterprise-changefeed-work.md %}) more frequently and deliver messages to the sink.
+
+Default: `3s`
+Recommendation: Decreasing to `1s`–`2s`.
+
+{{site.data.alerts.callout_danger}}
+Thoroughly test any decrease to the closed timestamp interval before deploying the change in production, because such an adjustment can cause [contention]({% link {{ page.version.version }}/performance-best-practices-overview.md %}#understanding-and-avoiding-transaction-contention) between transactions.
+{{site.data.alerts.end}}
+
+#### `kv.closed_timestamp.side_transport_interval`
+
+When there is a period of no write activity, the _side-transport_ is a node-level process that fixes a timestamp to close for groups of ranges. This cluster setting adjusts how often the side-transport process will attempt to move the [closed timestamp]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#closed-timestamps) forward. The smaller the interval, the more frequently the side-transport will attempt to move the timestamp forward.
+
+This will also affect changefeed checkpoints. That is, if the interval is decreased the changefeed will be able to read and checkpoint more frequently, rather than collecting changes into fewer, less frequent checkpoints. To improve changefeed latency, we recommend decreasing the `kv.closed_timestamp.side_transport_interval` cluster setting.
+
+For more information on the side-transport interval, refer to [An epic read on follower reads](https://www.cockroachlabs.com/blog/follower-reads-stale-data/) on the Cockroach Labs Blog.
+
```

Comment:
I think we should change our recommendation with respect to this setting.

Do not touch this setting.
Set the `kv.closed_timestamp_target_duration` to 3 seconds if it is 0 (that's the old default, which fell back to this setting).

It was a mistake (corrected on master, and backported, I think) to conflate these 2 settings together.  One is purely about rangefeed checkpoints.  This one IS about follower reads.  You certainly don't want to make your follower read queries slower because you're trying to tune changefeeds.

Reply:
I have actually removed the `side_transport_interval` setting with this change. It seems like we don't really want to talk about this here? Correct me though, if we do actually want to say something about it here.

Reply:
Yes; that's great.